client/src/components/doctorComponents/doctorForm/clinicInfo/ClinicInfo.jsx

```diff
@@ -61,6 +68,9 @@ function ClinicInfo({
                 setChange={setClinicName}
                 value={clinicName}
               />
+              {clinicNameErrorMsg.length !== 0 && (
+                <div className="error-msg">{clinicNameErrorMsg[0]}</div>
+              )}
             </div>
```

Comment:
You can use following to simplify since empty list evaluates to false. 
```
{clinicNameErrorMsg && (
  <div className="error-msg">{clinicNameErrorMsg[0]}</div>
)}
```
Also, name of a list is preferred to be plural. Consider switching to `clinicNameErrorMsgs`. Applies to all such blocks.